deployment/k3s/templates/deployments/calendar/postgres-volume.yaml

```diff
@@ -1,29 +1,29 @@
 apiVersion: v1
 kind: PersistentVolume
 metadata:
-  name: calendar-{{ $.Values.namespace }}
+  name: postgres
   labels:
     app: navigatum
-    deployment: maps
+    deployment: postgres
     type: local
 spec:
   storageClassName: local-path
   capacity:
     storage: 1Gi
-  persistentVolumeReclaimPolicy: Retain
+  persistentVolumeReclaimPolicy: Delete
```

Comment:
Does this mean a redeployment of Postgres removes all the data?

Reply:
No, if a volume would be deleted (i.e. manually), this means this volume is not Deleted instead of Retained. Given that this data is relatively eazy to recover, I think `Delete` is probably the better choice